src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -0,0 +1,120 @@
+export const description = `API Validation Tests for RenderPass Resolve.
+
+  Test Coverage:
+    - When resolveTarget is not null:
```

Comment:
Maybe test multiple attachment with different formats and each their resolve targets?

Reply:
Added a test that creates attachments resolving with some different format in the additional attachment slots.